### ISSUE TITLE
Web Developer starter site is not available in a specific instance

### ISSUE THREAD

**Issue by poonam279** [Codeinwp/templates-patterns-collection#327]
### Description

A user is unable to access the Web Developer starter site on a specific instance. He is using Neve Pro on multiple instances, and the starter site is accessible on those instances. I have attempted to use my own key in his instance, but the Web Developer starter site is not found. 

Is there any insight as to why this is happening?

Here is the staging site information:

Parrot Token: u#OaW*tAo-gSFpBbI
WordPress Login: https://mohammadarman.com/staging/wp-login.php
WordPress Version: 6.4.2
PHP Version: 8.0.30
Site Locale: en_US
Theme: Neve 3.7.4

### Step-by-step reproduction instructions

This is an edge case.

### Screenshots, screen recording, code snippet or Help Scout ticket

https://secure.helpscout.net/conversation/2453813119/396938?folderId=212385

### Is the issue you are reporting a regression

No

**Comment by vytisbulkevicius** [comment]
@poonam279,

I checked his website with provided credentials and I can see the Web Developer starter site:
<img width="917" alt="image" src="https://github.com/Codeinwp/templates-patterns-collection/assets/36594177/c63f9d23-6a43-4af2-a677-a4f72297f663">

Can you check again?

**Comment by poonam279** [comment]
@vytisbulkevicius I can also see it now. Before opening the issue I checked multiple times and it wasn't there. 
Can you please check on his main site? It's still not there. Screenshot: https://vertis.d.pr/i/uAvWv4
 Parrot Token: u#OaW*tAo-gSFpBbI
WordPress Login: https://mohammadarman.com/wp-login.php

**Comment by vytisbulkevicius** [comment]
Yes, doesn't appear on this one for me either. @preda-bogdan, do you have ideas here what could be the problem? I don't see any errors in the console related to this

**Comment by preda-bogdan** [comment]
This was a transient caching issue, the `tiob_sites` transient needed to be deleted so that it regenerates, it would have updated by itself in 12h time, hence why the first website resolved itself.  I did this already for the provided website and the starter site is now available.

In the future if this happens for other clients you can use **Transient Manager** or any other similar plugin to delete `tiob_sites`, this should then regenerate itself with fresh data.

Let me know if there is anything else here, if not we can close this.
Thank you!

cc: @poonam279, @vytisbulkevicius
